dask/async.py

```diff
@@ -426,69 +426,69 @@ def get_async(apply_async, num_workers, dsk, result, cache=None,
         result_flat = set([result])
     results = set(result_flat)
 
-    dsk = dsk.copy()
-    for f in start_cbs:
-        f(dsk)
-
-    dsk, dependencies = cull(dsk, list(results))
+    try:
+        dsk = dsk.copy()
+        for f in start_cbs:
+            f(dsk)
```

Comment:
Should be negligible for start/finish stuff. I'd expect slightly less negligible for pre/post task, but it probably doesn't matter there either.